packages/checkbox/src/vaadin-checkbox.js

```diff
@@ -170,6 +170,14 @@ class Checkbox extends SlotLabelMixin(
     return this.$.noop;
   }
 
+  /** @protected */
+  ready() {
+    super.ready();
+
+    this.addController(new InputController(this));
```

Comment:
Following the Lit way, I would suggest defining controllers rather in the `constructor` method.

Reply:
Done.